### ISSUE TITLE
tpm2-pytss crashes with FAPI.quote

### ISSUE THREAD

**Issue by gaetanww** [tpm2-software/tpm2-pytss#255]
tpm2-pytss is trying to create a string from a null pointer when returning from Fapi_Quote. See below:

```
<ipython-input-6-56c42e72ab86> in <module>
----> 1 out = fapi.quote('HE/KEY', [0, 1, 2, 3, 4, 5, 6, 7, 8, 9 , 10])

/opt/zte_py/.venv/lib/python3.8/site-packages/tpm2_pytss/FAPI.py in quote(self, path, pcrs, quote_type, qualifying_data)
    953             bytes(ffi.unpack(get_dptr(signature, lib.Fapi_Free), signature_len[0])),
    954             ffi.string(get_dptr(pcr_log, lib.Fapi_Free)).decode(),
--> 955             ffi.string(get_dptr(certificate, lib.Fapi_Free)).decode(),
    956         )
    957 
RuntimeError: cannot use string() on <cdata 'char *' NULL>
```

To reproduce:
```python
from tpm2_pytss import *
tcti = 'swtpm'
FAPIConfig(profile_name='P_ECCP256SHA256', tcti=tcti, ek_cert_less='yes')
fapi = FAPI()
fapi.provision()
fapi.create_key('HE/KEY', 'system, sign, restricted')
out = fapi.quote('HE/KEY', [0, 1, 2, 3, 4, 5, 6, 7, 8, 9 , 10])
fapi.close()
```

**Comment by whooo** [comment]
Can you test with the fix in https://github.com/tpm2-software/tpm2-pytss/pull/258 ?

**Comment by williamcroberts** [comment]
@gaetanww any update, we merged the fixes would like confirmation before closing the ticket.

**Comment by gaetanww** [comment]
@williamcroberts I tested and discussed the fix in #258. No crash for me anymore!
